app/views/shared/_select_work_type_modal.html.erb

```diff
@@ -7,6 +7,7 @@
       </div>
       <form class="new-work-select">
         <div class="modal-body">
+          <% params[:add_works_to_collection] = params['id'] if params['id'] && params['controller'] == 'hyrax/dashboard/collections' %>
```

Comment:
Why was this added? An inline comment would be helpful to understand. I don't usually expect that the `params` object is updated in the view. I'm wondering if this is better done somewhere else?

Reply:
I agree this is not optimal.  I was attempting to make the minimal change instead of refactoring.  Bad rush job.  I will look at refactoring now.

The thing I really don't like about the current code (at master) is that this file is in views/shared and is included on every page.  As far as I know, the selector is only needed in places where works get created.

* Dashboard -> My/All Works
* Collection -> show page
* maybe from the edit work form
* any place else?